app/controllers/miq_policy_controller.rb

```diff
@@ -322,6 +322,18 @@ def fetch_log
                        :message => _("Policy log downloaded"))
   end
 
+  # Not necessarily a whole id, but the kind of lowerCamelCase we use for model names
+  # within ids like 'po_xx-control_xx-control-containerImage'.
+  def ui_model_from_id(id)
+    return nil if id.nil?  # TODO: fix callers, remove
```

Comment:
See PR description.  I noticed some places are currently sometimes calling ui_lookup with nil.
That's a bug, but it that just returns nil which omits the word, while doing `nil.camelize` would crash, hence this guard.

#9401 a user-visible one I found, and I think there were more where the result gets overwritten by more specific functions.

I'm removing my change from iterate_status affecting get_json — yaml has CamelCase class names, not sure why I touched that.